src/Models/UserOrganizationMembership.php

```diff
@@ -25,8 +31,18 @@ public function __construct($attributes = null, array $options = [])
     {
         parent::__construct($attributes, $options);
         $this->user = $options['collection']->getUser();
-        // @TODO: Follow this dependency chain and invert it.
-        $this->organization = new Organization($attributes->organization);
-        $this->organization->memberships = [$this,];
+        $this->organization_info = $attributes->organization;
```

Comment:
If we're going this way, we can just leave it be and then `$this->get('organization')` inside of getOrganization.